### DISCUSSION TITLE
Could you please Anyone help Me I'm facing an issue with github it was while i'm entering my password it appears like this.

### DISCUSSION THREAD

**Discussion by RaviPs22** [community/community#70449] [New to GitHub]
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/RaviPs22/export-import-An-Array.git/'

**Comment by thevindu-w** [comment]
Use a personal access token instead of the account password. You can create one here: [https://github.com/settings/tokens](https://github.com/settings/tokens)

However, if you are trying to clone the repository [https://github.com/RaviPs22/export-import-An-Array.git/](https://github.com/RaviPs22/export-import-An-Array.git/), you don't need to enter any password or token as the repo is public. Make sure you entered the correct command.
```
git clone https://github.com/RaviPs22/export-import-An-Array.git
```

**Comment by Shailaputri** [comment]
Please follow these steps:-
1. Click on your profile photo on the navigation bar at the top of this page.
2. Go to settings as shown : 
<img width="188" alt="Screenshot 2023-10-18 at 7 06 33 PM" src="https://github.com/community/community/assets/75516191/70ff2cf4-3a21-410d-a435-237781f53855">

4. Scroll down. Towards the left click on Developer settings :
<img width="314" alt="Screenshot 2023-10-18 at 7 07 07 PM" src="https://github.com/community/community/assets/75516191/dc3073e9-c40a-46b2-a998-2faf37d75474">

5. Click on Tokens
<img width="1141" alt="Screenshot 2023-10-18 at 7 22 58 PM" src="https://github.com/community/community/assets/75516191/a4bc1b3a-f945-4b07-a715-83fca5737314">
(classic)

6. Click on Generate New Token
<img width="836" alt="Screenshot 2023-10-18 at 7 23 49 PM" src="https://github.com/community/community/assets/75516191/fb01b92c-81bc-4b97-8c72-65580e4e7c51">
7. github will ask for your password. 
8. After giving password, you will be redirected to this page. Here : select repo, workflow and project in case you want to contribute to public repos. Scroll to bottom and select 'Generate Token'. Save this token.
9. Now when you are asked for password, enter this token. 
You are good to go. Hope this helps.
